src/ncp/ncp_base.cpp

```diff
@@ -2173,6 +2177,24 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_DEBUG_NCP_LOG_LEVEL>(
     return error;
 }
 
+#if !OPENTHREAD_RADIO || OPENTHREAD_CONFIG_RADIO_CHANNEL_MASK
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_PHY_CHAN_SUPPORTED>(void)
```

Comment:
How about this for getter for `SPINEL_PROP_PHY_CHAN_SUPPORTED`?
- On FTD and MTD, we encode `otLinkGetSupportedChannelMask()`
- On RADIO, we encode `otPlatRadioGetSupportedChannelMask()`